tests/includes/phpunit7/speed-trap-listener.php

```diff
@@ -0,0 +1,354 @@
+<?php
+
+/**
+ * A PHPUnit TestListener that exposes your slowest running tests by outputting
+ * results directly to the console.
+ */
+class SpeedTrapListener implements PHPUnit_Framework_TestListener
+{
+
+    /**
+     * Internal tracking for test suites.
+     * Increments as more suites are run, then decremented as they finish. All
+     * suites have been run when returns to 0.
+     *
+     * @var integer
+     */
+    protected $suites = 0;
+
+    /**
+     * Time in milliseconds at which a test will be considered "slow" and be
+     * reported by this listener.
+     *
+     * @var int
+     */
+    protected $slow_threshold;
+
+    /**
+     * Number of tests to report on for slowness.
+     *
+     * @var int
+     */
+    protected $report_length;
+
+    /**
+     * Collection of slow tests.
+     *
+     * @var array
+     */
+    protected $slow = [];
+
+
+    /**
+     * Construct a new instance.
+     *
+     * @param array $options
+     */
+    public function __construct(array $options = [])
+    {
+        $this->loadOptions($options);
+    }
+
+
+    /**
+     * An error occurred.
+     *
+     * @param PHPUnit_Framework_Test $test
+     * @param Exception              $e
```

Comment:
Docblock refers to $e but method uses $t

Reply:
good eye...
this file is copied from:
https://github.com/WordPress/wordpress-develop/blob/master/tests/phpunit/includes/phpunit7/speed-trap-listener.php#L54-L57
so I'll just leave as is